### PR TITLE
Feature/bai 227 confirmation dialogue

### PR DESCRIPTION
**Pull request by ARADDCC011**
Added ConfirmationDialogue.tsx, and implemented it to confirm model deletion in [uuid].tsx, calling a dummy delete function

### CODE REVIEW

File: src/common/ConfirmationDialogue.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import Paper from '@mui/material/Paper'
+import Button from '@mui/material/Button'
+import Dialog from '@mui/material/Dialog'
+import DialogTitle from '@mui/material/DialogTitle'
+import DialogContent from '@mui/material/DialogContent'
+import DialogContentText from '@mui/material/DialogContentText'
+import DialogActions from '@mui/material/DialogActions'
+import { useState } from 'react'
+import { postEndpoint } from '../../data/api'
+import EmptyBlob from './EmptyBlob'
```
Comment (a3957273) [inline src/common/ConfirmationDialogue.tsx:12]: Lots of these imports aren't needed.

File: pages/model/[uuid].tsx [hunk 2]
```diff
@@ -178,6 +180,19 @@ function Model() {
...
+  const handleDelete = () => {
+    setDeleteConfirmOpen(true)
+  }
+
+  const handleDeleteCancel = () => {
+    setDeleteConfirmOpen(false)
+  }
+
+  const handleDeleteConfirm = () => {
+    console.log("Model would be deleted if this function did anything")
```
Comment (ARADDCC002) [inline pages/model/[uuid].tsx:192]: Remove log

**Review by a3957273 — CHANGES_REQUESTED** [review]

**Comment by ARADDCC002** [comment]
I think it would be worth adding a test for this component too.

**Review by ARADDCC002 — APPROVED** [review]

**Comment by ARADDCC002** [comment]
Closed as the work is being continued in #141

**Pull request closed without merging**